cmd/ctr/run_unix.go

```diff
@@ -92,6 +92,10 @@ func spec(id string, config *ocispec.ImageConfig, context *cli.Context, rootfs s
 	if tty {
 		env = append(env, "TERM=xterm")
 	}
+
+	// additional environment vars
+	env = append(env, context.StringSlice("env")...)
```

Comment:
What about `windows`? :angel: 

Although on windows I think the array actually need to only have the same variable defined once.